src/main/java/backend/github/UpdateIssuesTask.java

```diff
@@ -26,16 +27,21 @@ public UpdateIssuesTask(TaskRunner taskRunner, Repo repo, Model model) {
     public void run() {
         ImmutableTriple<List<TurboIssue>, String, Date> changes = repo.getUpdatedIssues(model.getRepoId(),
             model.getUpdateSignature().issuesETag, model.getUpdateSignature().lastCheckTime);
+        List<PullRequest> updatedPullRequests = repo.getUpdatedPullRequests(
+                model.getRepoId(), "", model.getUpdateSignature().lastCheckTime);
```

Comment:
Oh wait, I can just pass `""` to the super constructor all the time. I will change this then.